package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bat-client",
-  "version": "2.2.3",
+  "version": "2.2.4",
```

Comment:
You should run `npm install` after bumping this version so that it auto-updates the version in package-lock.json